addons/web/static/src/js/core/utils.js

```diff
@@ -500,6 +500,19 @@ var utils = {
     isComponent: function (value) {
         return value.prototype instanceof owl.Component;
     },
+    /**
+     * Checks if a keyboard event concerns
+     * the numpad decimal separator key.
+     *
+     * Some countries may emit a comma instead
```

Comment:
maybe complete the comment to include the link to the iso spec. That way we have a trace of why we made that choice